### PR TITLE
DEVX-2403: Validate orgId and projectId before attempting to install template

### PR DESCRIPTION
**Pull request by roribio**
## Related Issue

https://jira.corp.adobe.com/browse/DEVX-2403

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/commands/templates/install.js [hunk 1]
```diff
@@ -77,11 +77,11 @@ class InstallCommand extends BaseCommand {
...
-   * Setup Developer Console App Builder project from template install.yml configuration file
-   *
-   * @private
-   * @param {string} templateName
-   */
+     * Setup Developer Console App Builder project from template install.yml configuration file
+     *
+     * @private
+     * @param {string} templateName
+     */
```
Comment (slitviachenko) [inline src/commands/templates/install.js:84]: nitpick, the _previous_ annotations formatting was correct

File: src/commands/templates/install.js [hunk 2]
```diff
@@ -97,9 +97,13 @@ class InstallCommand extends BaseCommand {
...
     const appConfig = loadConfig({})
-    const orgId = appConfig.aio.project.org.id
-    const projectId = appConfig.aio.project.id
-    await templateManager.installTemplate(orgId, projectId)
+    const orgId = appConfig?.aio?.project?.org?.id
+    const projectId = appConfig?.aio?.project?.id
+    if (orgId && projectId) {
+      await templateManager.installTemplate(orgId, projectId)
+    } else {
+      aioLogger.error('Error installing template: Missing orgId or projectId in project configuration')
```
Comment (slitviachenko) [inline src/commands/templates/install.js:105]: Developers might miss error messages. I think we should stop processing a template installation throwing an exception. That way developers can rerun a template installation once the issue with orgId, projectId is fixed.

**Review by slitviachenko — CHANGES_REQUESTED** [review]

File: src/commands/templates/install.js (hunk 1 above)
Comment (roribio) [reply src/commands/templates/install.js]: Fixed ✅ This was a mixup with the linter, all good now.

File: src/commands/templates/install.js (hunk 2 above)
Comment (roribio) [reply src/commands/templates/install.js]: Fixed ✅

**Review by slitviachenko — APPROVED** [review]

**Pull request merged**